### ISSUE TITLE
ownerInject is not a function

### ISSUE THREAD

**Issue by tehmaestro** [offirgolan/ember-cp-validations#360]
Hi, I get the following error when trying to create an inline validator. I am slowly upgrading every addon/library/ember version, so this might be the issue, but I'm not sure how to solve it. I tried installing `ember-getowner-polyfill` but that didn't work. 

Could it be because of an older dependency, like ember-cli-shims version 0.0.3 ? Thanks.
vendor.js:91243 Uncaught TypeError: owner.ownerInjection is not a function

### Environment

- Ember Version: 1.12.2 
- Ember CLI Version: 0.2.3
- Ember CP Validations Version: 3.0.1

### Steps to Reproduce
Create a simple inline validator.

**Comment by offirgolan** [comment]
Hmm thats a weird issue. Ill take a look into this.

**Comment by offirgolan** [comment]
@tehmaestro Im really sorry but I'm not able to recreate this issue. Many of the test cases also frequently use inline validators and have no issue (tested on 1.11+). If you can create a simple dummy app to recreate the problem you are having, that would be really helpful.

**Comment by tehmaestro** [comment]
Hi, I suspect that doing a cleanup ( deleted node_modules, bower_components, tmp, reinstalled) fixed this. I did this things after upgrading to ember 1.13 and ember cli 1.13 and doing a cleanup + ember init. So I'm not sure that just cleaning up the project would have worked too (without updating to 1.13), but I think it would have. I got another error from another ember addon, and that one also went away after cleaning the project. Thanks anyways.
